conjure-go-client/httpclient/client_params_test.go

```diff
@@ -168,6 +170,135 @@ func TestBuilder(t *testing.T) {
 	}
 }
 
+func TestMiddlewareOrdering(t *testing.T) {
```

Comment:
Many of these tests fail on develop because parameters are applied in the "wrong" order for request mutations.